packages/integration-tests/projects/connect-popup/index.js

```diff
@@ -0,0 +1,370 @@
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+const path = require('path');
+
+const { Controller } = require('../../websocket-client');
+
+const defaults = {
+    // some random empty seed. most of the test don't need any account history so it is better not to slow them down with all all seed
```

Comment:
Is `connect-explorer` using in anyway the blockchain? I thought it was just and interface to talk to the device, but I might have missed something.

Reply:
this is setup for testing. I use connect-explorer to test connect-popup. And for that I need an initialized device